src/XMakeCommandLine/UnitTests/project.json

```diff
@@ -7,28 +7,25 @@
   },
   "frameworks": {
     "net451": {
-      "dependencies": {
-        "xunit.runner.visualstudio": "2.1.0"
-      }
+      "dependencies": {}
     },
     "net46": {
-      "dependencies": {
-        "xunit.runner.visualstudio": "2.1.0"
```

Comment:
From what I remember, @ValMenn said that MONO MSbuild would not compile because of a missing transitive dependency for xunit.runner.visualstudio under net46 on linux.

Since xunit test discovery within VS does not work on net core, we concluded that the easiest way to get unstuck is to remove it. 

We can work on getting it back up when test discovery works on net core (hopefully, the whole transitive dependency closure will also be available cross-platform by then). I can file an issue to add this back later.